### ISSUE TITLE
How to use codec AMR in Octet Aligned Mode ?

### ISSUE THREAD

**Issue by hamidrhashmi** [signalwire/freeswitch#683]
Is there any option in Configuration to enable "Octet-Aligned" Mode? 

```
<configuration name="amrwb.conf">
  <settings>
    <param name="default-bitrate" value="8"/>
    <param name="volte" value="0"/>
    <param name="adjust-bitrate" value="0"/>
    </settings>
</configuration>
```

and as per file mod_amrwb.c it depends on enum "AMRWB_OPT_OCTET_ALIGN" but how to set its value to 1 from amrwb.conf.xml ?

Moreover, when I see supported codecs, Freeswitch shows me two different codecs of AMR-WB
```
codec,AMR-WB / Bandwidth Efficient,mod_amrwb
codec,AMR-WB / Octet Aligned,mod_amrwb
```

How to offer AMR-WB / Octect Aligned instead of Bandwidth Efficient?

**Comment by hamidrhashmi** [comment]
@dragos-oancea thank you for the quick fix. But there is an issue that FreeSWITCH is running on a docker container, can I just compile mod_amrwb separately and move mod_amrwb.so file to the container and restart service? how can I just compile a module independently?

**Comment by dragos-oancea** [comment]
You could compile inside the container.

**Comment by crienzo** [comment]
Resolved by #684
